notifications/src/main/java/dk/shape/games/notifications/presentation/viewmodels/settings/NotificationsSettingsSwitcherViewModel.kt

```diff
@@ -8,38 +8,42 @@ import me.tatarka.bindingcollectionadapter2.BR
 import me.tatarka.bindingcollectionadapter2.ItemBinding
 import me.tatarka.bindingcollectionadapter2.itembindings.OnItemBindClass
 
-sealed class NotificationsSettingsSwitcherViewModel {
+class NotificationsSettingsSwitcherViewModel {
     val item: ObservableField<Any> = ObservableField(Loading)
 
     val itemBinding: ItemBinding<Any> = ItemBinding.of(
-        OnItemBindClass<Any>().map(
-            Loading::class.java,
-            BR.viewModel,
-            R.layout.view_notifications_switcher_loading
-        ).map(
-            Empty::class.java,
-            BR.viewModel,
-            R.layout.view_notifications_switcher_empty
-        ).map(
-            Error::class.java,
-            BR.viewModel,
-            R.layout.view_notifications_switcher_error
-        ).map(
-            Content::class.java,
-            BR.viewModel,
-            R.layout.view_notifications_switcher_content
-        )
+        OnItemBindClass<Any>()
+            .map(
+                Loading::class.java,
+                BR.viewModel,
+                R.layout.view_notifications_switcher_loading
+            )
+            .map(
+                Empty::class.java,
+                BR.viewModel,
+                R.layout.view_notifications_switcher_empty
+            )
+            .map(
+                Error::class.java,
+                BR.viewModel,
+                R.layout.view_notifications_switcher_error
+            )
+            .map(
+                Content::class.java,
+                BR.viewModel,
+                R.layout.view_notifications_switcher_content
+            )
     )
 
     private val subjectViewModels: List<NotificationsSettingsSubjectViewModel>?
-        get() = (item.get() as? Content)?.items?.filterIsInstance<NotificationsSettingsSubjectViewModel>()
+        get() = (item.get() as? Content?)?.items?.filterIsInstance<NotificationsSettingsSubjectViewModel>()
```

Comment:
do we actually need the questionmark for Content, since we already have it on the as?
```suggestion
        get() = (item.get() as? Content)?.items?.filterIsInstance<NotificationsSettingsSubjectViewModel>()
```